src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/customers/CustomerSyncIT.java

```diff
@@ -0,0 +1,147 @@
+import static com.commercetools.sync.integration.commons.utils.CustomerITUtils.createSampleCustomerJohnDoe;
+import static com.commercetools.sync.integration.commons.utils.CustomerITUtils.deleteCustomerSyncTestData;
+import static com.commercetools.sync.integration.commons.utils.ITUtils.createCustomFieldsJsonMap;
+import static com.commercetools.sync.integration.commons.utils.SphereClientUtils.CTP_SOURCE_CLIENT;
+import static com.commercetools.sync.integration.commons.utils.SphereClientUtils.CTP_TARGET_CLIENT;
+import static com.commercetools.sync.integration.commons.utils.StoreITUtils.createStore;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CustomerSyncIT {
+    private List<String> errorMessages;
+    private List<Throwable> exceptions;
+    private CustomerSync customerSync;
+
+    @BeforeEach
+    void setup() {
+        deleteCustomerSyncTestDataFromProjects();
+
+        createSampleCustomerJohnDoe(CTP_SOURCE_CLIENT);
+        createSampleCustomerJaneDoe(CTP_SOURCE_CLIENT);
+
+        createSampleCustomerJohnDoe(CTP_TARGET_CLIENT);
+
+        setUpCustomerSync();
+    }
+
+    @AfterAll
+    static void tearDown() {
+        deleteCustomerSyncTestDataFromProjects();
+    }
+
+    private static void deleteCustomerSyncTestDataFromProjects() {
+        deleteCustomerSyncTestData(CTP_SOURCE_CLIENT);
+        deleteCustomerSyncTestData(CTP_TARGET_CLIENT);
+    }
+
+    private void setUpCustomerSync() {
+        errorMessages = new ArrayList<>();
+        exceptions = new ArrayList<>();
+        final CustomerSyncOptions customerSyncOptions = CustomerSyncOptionsBuilder
+            .of(CTP_TARGET_CLIENT)
+            .errorCallback((exception, oldResource, newResource, actions) -> {
+                errorMessages.add(exception.getMessage());
+                exceptions.add(exception);
+            })
+            .build();
+        customerSync = new CustomerSync(customerSyncOptions);
+    }
+
+    @Test
+    void sync_WithoutUpdates_ShouldReturnProperStatistics() {
+
+        final List<Customer> customers = CTP_SOURCE_CLIENT
+            .execute(buildCustomerQuery())
+            .toCompletableFuture()
+            .join()
+            .getResults();
+
+        final List<CustomerDraft> customerDrafts = mapToCustomerDrafts(customers);
+
+        final CustomerSyncStatistics customerSyncStatistics = customerSync
+            .sync(customerDrafts)
+            .toCompletableFuture()
+            .join();
+
+        assertThat(errorMessages).isEmpty();
+        assertThat(exceptions).isEmpty();
+
+        assertThat(customerSyncStatistics).hasValues(2, 1, 0, 0);
+        assertThat(customerSyncStatistics
+            .getReportMessage())
+            .isEqualTo("Summary: 2 customers were processed in total (1 created, 0 updated and 0 failed to sync).");
+    }
+
+    @Test
+    void sync_WithUpdates_ShouldReturnProperStatistics() {
```

Comment:
Do you want to check if customer was updated correctly or you have this already in a different test?